docroot/themes/humsci/su_humsci_theme/js/hover_menu.js

```diff
@@ -8,12 +8,16 @@
       this.menuEdgeCheck(context);
 
       // Open/close the menu from hamburger button.
-      $header.find('button.fa-bars').once().click(function () {
+      $('button.fa-bars', $header).once().click(function () {
```

Comment:
Header is the whole dom each time. You are using `once()` here to prevent against multiple event adding. `context` should do this for you no?

Reply:
good point. i heard that

Reply:
i added `context` to header and removed `once()`